app/presenters/hyku/work_show_presenter.rb

```diff
@@ -53,6 +55,28 @@ def user_can_feature_collection?
     end
     # End Featured Collections Methods
 
+    # @return [Boolean] render a IIIF viewer
+    def iiif_viewer?
+      representative_id.present? &&
+        representative_presenter.present? &&
+        iiif_media? &&
+        Hyrax.config.iiif_image_server? &&
+        members_include_viewable?
+    end
+
+    def iiif_media?
```

Comment:
This method and an aspect of members_include_viewable are related.

Do we need to test if the user can read the representative?  I don't think we do.